### ISSUE TITLE
pm4py.format_dataframe raises an error when there are multiple timestamp columns

### ISSUE THREAD

**Issue by jonathan-brock** [process-intelligence-solutions/pm4py#254]
Hi everyone, 
when formatting a dataframe with pm4py.format_dataframe I get the error "DataFrame object has no attribute dtype."

I think it has to do with DataFrames where there are more than 2 timestamp columns (as shown in the screenshot below, in line 158 of the error)
df[col].dtype gets called on a DataFrame instead of a list hence the error.

Steps to reproduce are shown in the attached screenshot. 
![grafik](https://user-images.githubusercontent.com/47773832/132243432-9d065044-8da0-4835-a734-3a472fab67da.png)

**Comment by fit-alessandro-berti** [comment]
Dear jonathan-brock, the error is very strange, except if you assigned to a column an entire dataframe (or the name of the column is duplicated among the dataframe)

Could you print the output of the command dataframe.info() ?

**Comment by jonathan-brock** [comment]
Good Morning Alessandro,
you can find the running-example.csv on your PM4PY homepage [here](https://pm4py.fit.fraunhofer.de/static/assets/data/getting_started/running-example.csv), so you can reproduce the issure yourself.
dataframe.info() gives:
```
<class pandas.core.frame.DataFrame>
RangeIndex: 42 entries, 0 to 41
Data columns (total 6 columns):
 #   Column          Non-Null Count  Dtype 
---  ------          --------------  ----- 
 0   case_id         42 non-null     int64 
 1   activity        42 non-null     object
 2   timestamp       42 non-null     object
 3   costs           42 non-null     int64 
 4   resource        42 non-null     object
 5   time:timestamp  42 non-null     object
```

**Comment by fit-alessandro-berti** [comment]
Dear jonathan, thanks for signaling.

We were able to replicate the issue, that is due to Pandas duplicating columns when the format_dataframe is applied

We will release a fix for the issue in a future release
For the moment, i think you can keep your code but commenting the df["time:timestamp"] = df["timestamp"] row

Have a nice day
Alessandro

**Comment by fit-alessandro-berti** [comment]
Dear jonathan, we resolved the issue in the current release 2.2.13.1

Thanks for signaling and have a nice day
